README.md

```diff
@@ -28,7 +28,9 @@ Examples:
 
 Installation
 ------------
-composer require "moc/redirects" "1.0.*"
+The package can be easily installed by using composer - just add the following line to your `composer.json` file:
```

Comment:
Hey Mario. Thanks for the PR, however that's not really correct.. run the following command to require the package, not add the line to the composer.json file